two_factor/middleware/enforce2fa.py

```diff
@@ -0,0 +1,21 @@
+"""Middleware to check if staff user has 2FA unabled."""
+
+from django.shortcuts import redirect
+from django.urls import resolve, reverse
+
+
+class Enforce2FAMiddleware:
```

Comment:
This package still supports Python 2.7 for Django 1.11, so that should be a subclass of `object`.